packages/client/src/session-manager.test.ts

```diff
@@ -5,6 +5,7 @@ import { SessionStorage } from './storage';
 const transaction = {
   codeVerifier: 'codeVerifier',
   redirectUri: 'redirectUri',
+  state: 'state',
```

Comment:
it's hard to maintain the content of the `transaction`.
let's set the `transaction` type to `Session` explicitly in case someone forget change it in the future.

Reply:
Fixed.